README.md

```diff
@@ -76,6 +76,7 @@ Any of the previous execution methods accept configuration through the following
 | `GITBASEPG_PORT` | `--port` | `8080` | Port to bind the HTTP server |
 | `GITBASEPG_SERVER_URL` | `--server` | | URL used to access the application in the form `HOSTNAME[:PORT]`. Leave it unset to allow connections from any proxy or public address |
 | `GITBASEPG_DB_CONNECTION` | `--db` | `root@tcp(localhost:3306)/none?maxAllowedPacket=4194304` | gitbase connection string. Use the DSN (Data Source Name) format described in the [Go MySQL Driver docs](https://github.com/go-sql-driver/mysql#dsn-data-source-name). |
+| `GITBASEPG_CONN_MAX_LIFETIME` | `--conn-max-lifetime` | `30` | Connections max life time since their creation in seconds |
```

Comment:
Let's add a mention to the timeout config in gitbase. Something like
```suggestion
| `GITBASEPG_CONN_MAX_LIFETIME` | `--conn-max-lifetime` | `30` | Maximum amount of time a SQL connection may be reused, in seconds. Make sure this value is lower than the timeout configured in the gitbase server, set with [`GITBASE_CONNECTION_TIMEOUT`](https://docs.sourced.tech/gitbase/using-gitbase/configuration#environment-variables) |
```

Reply:
Is it reasonable for this web UI to wait for more than a couple of minutes?
On the other hand, if that's the use case of this playground, let's do it.
@smacker do you remember an average of your slow queries?

Reply:
@smacker that's a nice point, I forgot to tell that the chosen default was not inspired by any particular thought. Anyway the fact that some queries are taking longer shouldn't affect this value as non-idle connection can still be in-use after the timeout. It just cannot be started after the timeout. At least this is what I read from [here](https://www.alexedwards.net/blog/configuring-sqldb).

> A connection can still be in use more than <timeout value> after being created — it just cannot start to be reused after that time.

Reply:
oh, nice! Thank you for the clarification! 👍